README.md

```diff
@@ -179,6 +179,22 @@ postgres=> \dx pg_cron
 (1 row)
 ```
 
+## Managing Policies Without Table Ownership
+
+In Postgres, only table owners can create RLS policies for a table. This can be
+limiting if you need to allow certain roles to manage policies without allowing
+them to perform other DDL (e.g. to prevent them from dropping the table).
+
+With supautils, this can be done like so:
+
+```
+supautils.privileged_role = 'my_privileged_role'
+supautils.privileged_role_allow_policies_on_tables = 'public.not_my_table, public.also_not_my_table'
+```
+
+This allows `my_privileged_role` to manage policies for `public.not_my_table`
+and `public.also_not_my_table` without being an owner of these tables.
+
```

Comment:
Thinking more about this. What you really want is something like `GRANT CREATE POLICY ON TABLE TO <role>`. Maybe this feature can be done with a single config.

Reply:
Yup exactly, ideally `POLICY` can be a table privilege, that way we can decouple it from table ownership.

This is already a single config though, the `privileged_role` is a config we already use for other things (e.g. `privileged_role_allowed_configs`)

Reply:
Ok, cool. So since supautils adds security mechanisms that aren't in postgres yet, I think the proper design for this is:

```sql
supautils.grant_create_policy = '{"role1": ["table1", "table2"]}'
```

This way it's a generic solution and not just Supabase-specific. 

(I know it's not exactly GRANT CREATE POLICY since it's global but it's the best we can do without creating tables)

Also done this way later:

- We won't cause a breaking change
- We can use this flexibility if the need arises